.eslintignore

```diff
@@ -0,0 +1,2 @@
+node_module
+packages/@noma/helper-env/src/index.js
```

Comment:
Had to ignore this file because it is using top level await which eslint does not support yet